src/test/java/org/jabref/logic/importer/util/FileFieldParserTest.java

```diff
@@ -150,12 +150,6 @@ private static Stream<Arguments> stringsToParseTest() throws Exception {
                         "::file.pdf"
                 ),
 
-                // tooManySeparators
```

Comment:
(I just scrolled through)

Why is this test removed? "too many separators" should be kept as test (shouldn't it)?

Reply:
You are right, I could have moved it by one. When making the change it seemed to me that it was colliding with my code but checking for too many separators is something that can still be done. I will revert this and update it to fix it.

Reply:
Thank you!

Maybe, even more tests can be added. Without mimetype (thid parameter), without description.

Reply:
I have added back this test, with the change to make it work with the new field format, and included a few more tests as you suggested.